apps/client/src/sitemap.xml

```diff
@@ -0,0 +1,303 @@
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/macro-translator</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/profile</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/island-workshop</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/island-animals</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/about</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/import</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/patreon-redirect</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/tipeee-redirect</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/list-import/:importString</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/custom-links</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/link</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/template</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/desynth</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/retainers</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/retainer-ventures</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/voyage-tracker</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
+    <url>
+        <loc>https://ffxivteamcraft.com/db</loc>
+        <priority>0.80</priority>
+        <lastmod>2024-06-02T14:33:09.125Z</lastmod>
+    </url>
```

Comment:
The db module has many child pages, this would probably be around 2M pages with all the content that's provided, but this path won't work.